### PR TITLE
[WIP] Frio: restyle login page

### PR DESCRIPTION
**Pull request by fabrixxm**
Restyle login page in Frio theme

### CODE REVIEW

File: view/theme/frio/config.php [hunk 1]
```diff
@@ -75,6 +76,7 @@ function theme_admin(App $a) {
 	$arr["contentbg_transp"] = Config::get('frio', 'contentbg_transp');
 	$arr["background_image"] = Config::get('frio', 'background_image');
 	$arr["bg_image_option"]  = Config::get('frio', 'bg_image_option');
+    $arr["login_bg_image"]   = Config::get('frio', 'login_bg_image');
```
Comment (MrPetovan) [inline view/theme/frio/config.php]: Standards: Wrong indentation, please use tabs instead.

File: view/theme/frio/config.php [hunk 2]
```diff
@@ -104,7 +106,7 @@ function frio_form($arr) {
 	$background_image_help = "<strong>" . t("Note"). ": </strong>".t("Check image permissions if all users are allowed to visit the image");
 
 	$t = get_markup_template('theme_settings.tpl');
-	$o .= replace_macros($t, array(
+    $ctx = array(
```
Comment (MrPetovan) [inline view/theme/frio/config.php]: Standards: Wrong indentation, please use tabs instead.

File: view/theme/frio/config.php [hunk 3]
```diff
@@ -116,7 +118,13 @@ function frio_form($arr) {
 		'$contentbg_transp' => array_key_exists("contentbg_transp", $disable) ? "" : array('frio_contentbg_transp', t("Content background transparency"), ((isset($arr["contentbg_transp"]) && $arr["contentbg_transp"] != "") ? $arr["contentbg_transp"] : 100)),
 		'$background_image' => array_key_exists("background_image", $disable ) ? "" : array('frio_background_image', t('Set the background image'), $arr['background_image'], $background_image_help),
 		'$bg_image_options' => Image::get_options($arr),
-	));
+	);
+
+    if ( array_key_exists("login_bg_image", $arr ) &&  !array_key_exists("login_bg_image", $disable ) ) {
```
Comment (MrPetovan) [inline view/theme/frio/config.php]: Standards: Wrong indentation, please use tabs instead.

File: view/theme/frio/php/default.php [hunk 4]
```diff
@@ -54,9 +54,12 @@
 	}
 	echo '
 		<meta name="theme-color" content="' . $nav_bg . '" />';
+
+    $is_singleuser = Config::get('system','singleuser');
```
Comment (MrPetovan) [inline view/theme/frio/php/default.php]: Standards: Wrong indentation, please use tabs instead.

File: view/theme/frio/style.php [hunk 5]
```diff
@@ -42,6 +42,7 @@
 		$contentbg_transp = Config::get("frio", "contentbg_transp");
 		$background_image = Config::get("frio", "background_image");
 		$bg_image_option  = Config::get("frio", "bg_image_option");
+        $login_bg_image   = Config::get("frio", "login_bg_image");
```
Comment (MrPetovan) [inline view/theme/frio/style.php]: Standards: Wrong indentation, please use tabs instead.

File: view/theme/frio/style.php [hunk 6]
```diff
@@ -175,6 +177,7 @@
 	'$contentbg_transp'            => $contentbg_transp,
 	'$background_image'            => $background_image,
 	'$background_size_img'         => $background_size_img,
+    '$login_bg_image'              => $login_bg_image,
```
Comment (MrPetovan) [inline view/theme/frio/style.php]: Standards: Wrong indentation, please use tabs instead.

**Comment by fabrixxm** [comment]
didn't we decided for spaces? :-) :-P

**Comment by annando** [comment]
Yes, we didn't 😝

**Comment by foss-** [comment]
Hope it's ok, to add some feedback on this change in this PR. Always unsure wether to use this or the developers group on friendica.

@fabrixxm Thanks for working on this. Are you aware of https://github.com/friendica/friendica/issues/3671 ? This partly addresses some of the issues mentioned.

Now that the login is available by default, can we remove the login icon at the top right? It opens a login form in an overlay over the login form already shown.
new loginpage

Looking at this page on a public instance e.g. https://libranet.de/ there would still be the need to add a "Sign up" button to the login form. Having such a button would allow to get rid of both top right icons for Sign Up and Login.

There's a styling problem with the way the text is displayed. The contrast is too low making it very hard to read.
text contrast

And yes, the page looks nice. I really do like the overall design. Happy to see this important place see some love. In the end it's the first page new users see and then decide if they signup or not. So making a good first impression is critical in order to attract new users.

**Comment by fabrixxm** [comment]
Yup, the PR was WIP, I'm planning to work on it more as soon as I find some time.

**Comment by rabuzarus** [comment]
> Now that the login is available by default, can we remove the login icon at the top right? It opens a login form in an overlay over the login form already shown.

From a logical point of view we can only remove the login icon from `/` and `/logout`. The other pages would need this button because you won't see the login form on this pages.

**Comment by AlfredSK** [comment]
Oh! Didn't know this new design is enabled by default.

**Comment by AlfredSK** [comment]
I'm missing the possibility to have no background picture on the login page at all. I can choose my own picture. But cannot set the login page to just use the background color without any picture.

**Comment by tobiasd** [comment]
> I'm missing the possibility to have no background picture on the
> login page at all. I can choose my own picture. But cannot set the
> login page to just use the background color without any picture.

You could try to use a " " (space) as filename of the BG image as workaround.

**Comment by fabrixxm** [comment]
@AlfredSK I'll add the option

**Comment by AlfredSK** [comment]
@tobiasd @fabrixxm Thank you. 

@foss- I added font color tags to my home.html as a workaround. Looks better now. :-)

**Pull request merged**